src/Compilers/CSharp/Portable/Symbols/Source/SourceEventAccessorSymbol.cs

```diff
@@ -15,17 +15,64 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     internal abstract class SourceEventAccessorSymbol : SourceMemberMethodSymbol
     {
         private readonly SourceEventSymbol _event;
+        private readonly string _name;
+        private readonly ImmutableArray<MethodSymbol> _explicitInterfaceImplementations;
```

Comment:
>ImmutableArray<MethodSymbol> [](start = 25, length = 28)

This appears to be an `ImmutableArray` that is either empty (when there is no corresponding implemented event) or contains a single method (when there is a corresponding implemented event).  If I am right, for this situation I would expect to see a field of type `MethodSymbol` that is either `null` or contains the implemented event's accessor.

Reply:
Ah, I see this implements an API that handles imported symbols as well, which can implement multiple methods.

---
In reply to: [286220692](https://github.com/dotnet/roslyn/pull/35756#discussion_r286220692) [](ancestors = 286220692)